### ISSUE TITLE
"Not Found" for all instances 

### ISSUE THREAD

**Issue by bradley2083** [dannyvankooten/edd-helpscout#42]
Hey, 

I just installed your plugin and followed the instructions to a T and I'm getting a "Not Found" message for all customer tickets within our HelpScout sidebar.  Any idea why this would  happen?  I've double and triple checked the secret key and that is not the issue.  I can share the debug log if that helps, just let me know.  Please note that all of our customers are currently on free trials and I know that "free trials" are relatively new in EDD, so wondering if that's part of the issue. 

Let me know what you need from me to help debug. 

Thank you!

**Comment by dannyvankooten** [comment]
Hi @bradley2083,

"Not found" would usually indicate that you made a mistake in your HelpScout integration settings. Please double check if it's pointing to your site correctly.

**Comment by bradley2083** [comment]
Hey @dannyvankooten,

Thanks for the quick reply.  The settings are all setup properly from what I can tell.  There are only the four pretty straightforward settings, right?

I have...

App Name: Easy Digital Downloads
Content Type: Dynamic Content
Callback URL:  https://MYSITE.com/edd-helpscout-api/customer_info  (MYSITE is using my domain)
Secret Key:  this is set to a 40-character string that matches what I have in my wp-config.php

Debug is toggled "off" 
Mailboxes - I have the appropriate mailbox checked out of the two mailboxes we have in HS. 

Other ideas?

**Comment by dannyvankooten** [comment]
Hey @bradley2083,

Yeah that looks about right, provided that https://MYSITE.com does resolve to your actual site? The "404 not found" is an error thrown by HelpScout and not by this plugin, so something in your set-up is causing HelpScout to never even reach / trigger the code in the plugin.

**Comment by bradley2083** [comment]
Hey @dannyvankooten 

When I try to visit the link that's in my callback URL (https://coursecats.com/edd-helpscout-api/customer_info), I get a 404.  Should this show some other page instead?  Other than that, the settings are all pretty simple and I can't see that there is anything else I can really adjust to get this to work.  Is there a way for me to send you login to our HelpScout setup?  

Thank you!

**Comment by dannyvankooten** [comment]
Hi Brad,

Are you sure you have the EDD HelpScout plugin loaded & activated? This is
what it looks like on a properly configured website when trying to access
it from your browser: https://mc4wp.com/edd-helpscout-api/customer_info .

Your HelpScout config is probably fine, but something on your WordPress
isn't running properly.

Additionally, please double check if your server is running PHP 5.3 since
the plugin will fail silently if you're on an older version.

--
Danny van Kooten
dannyvankooten.com

**Comment by bradley2083** [comment]
Yeah the plugin is active and we are running PHP 5.6. 
<img width="792" alt="screen shot 2016-11-29 at 9 03 18 am" src="https://cloud.githubusercontent.com/assets/1781226/20717485/d9d947d6-b612-11e6-92f0-dac47bf1adea.png">

**Comment by dannyvankooten** [comment]
Ahhhh, I see now. You installed the plugin from WordPress.org while this
GitHub repository references a newer version of the plugin (that is not yet
released to wordpress.org). The installation instructions for the older
version can be found here:
https://wordpress.org/plugins/edd-helpscout/installation/ .

If you follow the installation instructions on GitHub then you should also
be using the version from GitHub. Sorry for the hassle. :(

--
Danny van Kooten
dannyvankooten.com

**Comment by bradley2083** [comment]
Ha!  No worries, all good. I've uploaded the zip from the GitHub repo and it's now working.  Glad we got that figured out ;)

Thanks!

**Comment by bradley2083** [comment]
I actually just added this to another site of mine for our other HelpScout inbox and I'm getting an "Invalid Signature" notice in the HS sidebar - I've verified that my 40-character key matches in both HS and my wp-config.php file.  Any other ideas on what would cause this issue?  The only thing that is different on this site is that we have W3 Total Cache installed - everything else is essentially the same.
